tests/test_integration/test_stencil_object.py

```diff
@@ -34,11 +36,13 @@ def stencil(in_field: Field[float], out_field: Field[float], *, offset: float):
             )
 
     shape = (4, 4, 4)
-    in_storage = gt_storage.ones(
-        backend=backend, default_origin=(0, 0, 0), shape=shape, dtype=float
+    in_storage = OriginWrapper(
+        array=gt_storage.ones(backend=backend, aligned_index=(0, 0, 0), shape=shape, dtype=float),
+        origin=(0, 0, 0),
```

Comment:
To be clear, `OriginWrapper` etc are just utility wrappers for testing. There won't be anything user-faceing like that. I think it isn't worth to have additional API just to avoid that small repetition.